app/server/locale/ussd.sw.yml

```diff
@@ -74,20 +74,20 @@ sw:
         10. Onyesha chaguzi zilizopita
     account_management: |
       CON Akaunti yangu
-      1. Biashara Yangu
+      1. Wasifu wangu
       2. Chagua lugha utakayotumia
       3. Angalia salio
       4. Badilisha nambari ya siri
       5. Ondoka Sokoni
       0. Nyuma
-    my_business: |
-      CON Biashara Yangu
-      1. Angalia biashara
-      2. Badilisha biashara
-      0. Nyuma
-    about_my_business:
-      bio: END Unauza %{user_bio}
-      none: END Tafadhali enda kwa biashara yako ili ukaweze kuweka bidhaa unayouza
+    about_me:
+      profile: |
+        END Wasifu wako una maelezo yafuatayo:
+            Jina %{full_name}
+            Jinsia %{gender}
+            Eneo %{location}
+            Unauza %{user_bio}
+      none: END Tafadhali enda kwa wasifu wako ili ukaweze kuweka %{empty_attrs}.
```

Comment:
could it just go there? rather than show this message

Reply:
I think it would be a bit confusing and may present as a bug because consider the flow of getting there in the first place:
Start -> My account -> My profile -> See my profile
If we expound this My profile Menu it looks like this:
English:
```
      1. Edit name.
      2. Edit gender.
      3. Edit location.
      4. Change business.
      5. See my profile
      0. Back
```

Swahili:
```
      1. Weka jina.
      2. Weka Jinsia.
      3. Weka eneo.
      4. Badilish biashara.
      5. Angalia wasifu wako.
      0. Nyuma
```

So if you enter the see my profile key and not see some response related to that profile data, it might be disorienting.
I think its should stay as is because the user can already see the opportunities to enter data before even getting to the see profile section